public/js/combinationalAnalysis.js

```diff
@@ -101,8 +101,24 @@ function createBooleanPrompt(inputListNames,outputListNames,scope=globalScope){
             // //console.log(dataSample);
             drawCombinationalAnalysis(minmizedCircuit,inputListNames,outputListNames,scope)
         },
-
-        }
+      },
+      {
+        text: "Download",
+        click: function() {
+           var sTable = document.getElementById('combinationalAnalysis').innerHTML;
+           var style = "<style> table {font: 20px Calibri;} table, th, td {border: solid 1px #DDD;} padding: 2px 3px;text-align: center;} </style>";
+           var win = window.open('', '', 'height=700,width=700');
```

Comment:
I think this is a new browserTab. Maybe you can change the variable to **browserTab**